src/test/com/Elessar/app/util/KafkaTest.java

```diff
@@ -0,0 +1,105 @@
+package com.Elessar.app.util;
+
+import com.Elessar.app.util.kafka.ConsumerCreator;
+import com.Elessar.app.util.kafka.KafkaConstants;
+import com.Elessar.app.util.kafka.ProducerCreator;
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.NewTopic;
+import org.apache.kafka.clients.consumer.Consumer;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.clients.producer.RecordMetadata;
+import org.apache.kafka.common.serialization.StringDeserializer;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+/**
+ * Created by Hans on 3/17/19.
+ */
+public class KafkaTest {
+    private static final String testTopic = "Test";
```

Comment:
This is ok, but I was thinking we should just start writing the actual kafka sender and receiver and use them in this test.